interp/jit_stub.go

```
@@ -2,12 +2,14 @@
 const nativeBackend = false
 
 // newCompiler returns (nil, nil) on architectures without a native
 // backend. A nil compiler is the interpreter's signal that JIT is
 // unavailable, so callers gate on i.compiler == nil rather than an error.
-// lower is unreachable because compiler is never constructed here.
-func newCompiler() (*compiler, error) { return nil, nil }
+// newMachine is unreachable because compiler is never constructed here.
+func newCompiler() (*jit.Compiler, error) { return nil, nil }
 
-func lower(*lowering, plan) bool { return false }
+func newMachine() jit.Machine { return nil }
```

golangci-lint (2.12.2)

[error] func newMachine is unused

(unused)